apps/src/code-studio/pd/fit_weekend1819_registration/Confirmation.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import {
+  FormGroup,
+} from 'react-bootstrap';
+
+import LabeledFormComponent from '../form_components/LabeledFormComponent';
+
+export default class Confirmation extends LabeledFormComponent {
+  static associatedFields = [];
```

Comment:
It's used by FormController, and we technically do need it to be at least defined so that FormController can make some assumptions. But! With the new ES7 classes we could probably have it reasonably default to the empty array. I'll play around with that